go.mod

```diff
@@ -18,8 +18,7 @@ require (
 	github.com/dmetzgar/goveralls v0.0.3
 	github.com/eapache/go-resiliency v1.2.0 // indirect
 	github.com/emirpasic/gods v0.0.0-20190624094223-e689965507ab
-	github.com/fatih/color v0.0.0-20181010231311-3f9d52f7176a
-	github.com/fatih/structtag v1.1.0 // indirect
+	github.com/fatih/color v1.10.0
```

Comment:
(to pick a random version change)
these changes are the result of a `go get` for revive, not a `go get -u`, so it likely has these versions as minimums.
either way though, none of these seem particularly concerning.